subprojects/language-java/src/main/java/org/gradle/api/internal/tasks/compile/incremental/classpath/ClasspathEntrySnapshot.java

```diff
@@ -91,6 +90,16 @@ private DependentsSet affectedSince(ClasspathEntrySnapshot other) {
                 affected.addAll(dependents.getDependentClasses());
             }
         }
+        for (String added : addedSince(other)) {
+            if (added.endsWith("package-info")) {
```

Comment:
Can we reference the constant in `ClassSetAnalysisData` or put in in a commonly accessible place?

Reply:
Will do